31_HLSLPathTracer/app_resources/hlsl/pathtracer.hlsl

```diff
@@ -0,0 +1,320 @@
+#ifndef _NBL_HLSL_EXT_PATHTRACER_INCLUDED_
+#define _NBL_HLSL_EXT_PATHTRACER_INCLUDED_
+
+#include <nbl/builtin/hlsl/colorspace/EOTF.hlsl>
+#include <nbl/builtin/hlsl/colorspace/encodeCIEXYZ.hlsl>
+#include <nbl/builtin/hlsl/math/functions.hlsl>
+#include <nbl/builtin/hlsl/bxdf/bxdf_traits.hlsl>
+
+#include "rand_gen.hlsl"
+#include "ray_gen.hlsl"
+#include "intersector.hlsl"
+#include "material_system.hlsl"
+#include "next_event_estimator.hlsl"
+#include "scene.hlsl"
+
+namespace nbl
+{
+namespace hlsl
+{
+namespace ext
+{
+namespace PathTracer
```

Comment:
lower snake_case `path_tracing` we can also move it to core Nabla and not keep in `ext`